robottelo/api/utils.py

```diff
@@ -946,3 +946,26 @@ def skip_yum_update_during_provisioning(template=None, reverse=False):
         return update_provisioning_template(name=template, old=old, new=new)
     else:
         return update_provisioning_template(name=template, old=new, new=old)
+
+
+def hammer_api_timeout_set(timeout=-1, reverse=False):
```

Comment:
any reason keeping the timeout as -1

Reply:
-1 is for no timeout[1]  and could help in case not clear on exact time-out lengths 

[1] https://access.redhat.com/solutions/1274723